### PR TITLE
feat: Better transfer flow

### PR DESCRIPTION
**Pull request by ptbrowne**
- Show account icons
- Show account balances
- Password slide
- Show modals (WIP)

### CODE REVIEW

File: src/ducks/transfers/TransferPage.jsx [hunk 1]
```diff
@@ -109,9 +122,21 @@ const BeneficiaryRow = ({ beneficiary, onSelect }) => {
...
+            <Bold>
+              <Figure
+                symbol="€"
+                total={beneficiary.account.balance}
+                coloredPositive
+                coloredNegative
+                coloredWarning
+              />
+            </Bold>
+            )
```
Comment (drazik) [inline src/ducks/transfers/TransferPage.jsx]: These parenthesis makes it harder to read for me :(

File: src/ducks/transfers/TransferPage.jsx [hunk 2]
```diff
@@ -295,6 +297,34 @@ const _Summary = ({
 
 const Summary = translate()(_Summary)
 
+const TransferSuccess = translate()(({ t, onReset, onExit }) => (
+  <div>
+    {t('transfer.success.description')}
+    <br />
```
Comment (drazik) [inline src/ducks/transfers/TransferPage.jsx]: Nit: put the text inside a `<p>` and remove the `<br />`

File: src/ducks/transfers/TransferPage.jsx [hunk 3]
```diff
@@ -295,6 +297,34 @@ const _Summary = ({
...
+      onClick={onReset}
+      label={t('transfer.new-transfer')}
+    />
+  </div>
+))
+
+const TransferError = translate()(({ t, onReset, onExit }) => (
+  <div>
+    {t('transfer.error.description')}
+    <br />
```
Comment (drazik) [inline src/ducks/transfers/TransferPage.jsx]: Nit: put the text inside a `<p>` and remove the `<br />`

File: src/ducks/transfers/TransferPage.jsx [hunk 4]
```diff
@@ -303,7 +333,10 @@ class TransferPage extends React.Component {
       slide: 0,
       senderAccount: null,
       senderAccounts: [], // Possible sender accounts for chosen person
-      amount: ''
+      amount: '',
+      transferSent: false,
+      sendingTransfer: false,
+      transferError: null
```
Comment (drazik) [inline src/ducks/transfers/TransferPage.jsx]: Nit: I think using a string and a single property would be better, so impossible states are impossible ©

File: src/components/PercentageLine.styl [hunk 5]
```diff
@@ -1,2 +1,3 @@
 .PercentageLine
-    transition width 0.3s ease
+    transition transform 0.3s ease
```
Comment (drazik) [inline src/components/PercentageLine.styl]: <3

File: src/components/BottomButton.jsx [hunk 6]
```diff
@@ -4,22 +4,34 @@ import { Button } from 'cozy-ui/react'
...
+    ...buttonStyle,
+    opacity: 1,
+    transform: `translateY(0px)`
+  },
+  invisibleButton: {
+    ...buttonStyle,
+    opacity: 0,
+    transform: `translateY(100px)`
+  }
+}
```
Comment (drazik) [inline src/components/BottomButton.jsx]: Why didn't you extract these to a css file ?

File: src/ducks/transfers/TransferPage.jsx (hunk 4 above)
Comment (ptbrowne) [reply src/ducks/transfers/TransferPage.jsx]: 💯

File: src/components/BottomButton.jsx (hunk 6 above)
Comment (ptbrowne) [reply src/components/BottomButton.jsx]: I am tired of writing CSS files for such tiny components... :)

File: src/ducks/transfers/TransferPage.jsx (hunk 4 above)
Comment (ptbrowne) [reply src/ducks/transfers/TransferPage.jsx]: Adding a comment and solving this in a later PR.

File: src/ducks/transfers/TransferPage.jsx (hunk 3 above)
Comment (ptbrowne) [reply src/ducks/transfers/TransferPage.jsx]: Later PR will deal with modal styles, thanks !

File: src/ducks/transfers/TransferPage.jsx (hunk 1 above)
Comment (ptbrowne) [reply src/ducks/transfers/TransferPage.jsx]: Eagle eye ! Thx

**Pull request merged**